src/pubtools/_marketplacesvm/cloud_providers/base.py

```diff
@@ -298,8 +298,8 @@ def delete_push_images(
         Returns:
             object: The publish result data.
```

Comment:
Can you update info in documentation as well?

Reply:
Good catch, thanks!

Fixed